sound.js

```diff
@@ -53,6 +53,9 @@ Sound.prototype.isLoaded = function() {
 Sound.prototype.play = function(onEnd) {
   if (this._loaded) {
     RNSound.play(this._key, (successfully) => onEnd && onEnd(successfully));
+  } else {
+    console.warn('Tried to play sound that was not yet loaded.');
```

Comment:
Can you remove this line? In the onEnd callback, the client can decide whether and how to log.